plugins/roundedwindow/roundedwindow.cpp

```diff
@@ -239,7 +237,7 @@ bool RoundedWindow::supported()
     if (desktop.isEmpty())
         return false;
 
-    return desktop == "Lingmo" && KWin::effects->isOpenGLCompositing() && KWin::GLRenderTarget::supported();
+    return desktop == "Lingmo" && KWin::effects->isOpenGLCompositing() && KWin::GLFramebuffer::supported();
```

Comment:
Yes, this makes sense  that KDE have renamed this API. Consider Checking the version of kWin to determine which API to use.

See https://invent.kde.org/plasma/kwin/-/merge_requests/2231

Reply:
Such as checking ` #if KWIN_EFFECT_API_VERSION < 233`

Reply:
> Such as checking ` #if KWIN_EFFECT_API_VERSION < 233`

I prefer limit the minimum API version.
Another problem is, this project is hyper-dependent to `lingmo-kwin`, which also means this project should follow the API from `lingmo-kwin`. 
It's already a customized package, compile the package with origin kwin was confirmed to make extra errors.
Anyways, I will check it out. :D